x-pack/plugins/security_solution/public/detections/components/rules/eql_query_bar/eql_query_bar.tsx

```diff
@@ -47,13 +85,17 @@ export const EqlQueryBar: FC<EqlQueryBarProps> = ({ dataTestSubj, field, idAria
       data-test-subj={dataTestSubj}
       describedByIds={idAria ? [idAria] : undefined}
     >
-      <EuiTextArea
-        data-test-subj="eqlQueryBarTextInput"
-        fullWidth
-        isInvalid={isInvalid}
-        value={fieldValue}
-        onChange={handleChange}
-      />
+      <>
+        <TextArea
+          data-test-subj="eqlQueryBarTextInput"
+          fullWidth
+          isInvalid={isInvalid}
+          value={fieldValue}
+          onBlur={handleValidation}
+          onChange={handleChange}
+        />
```

Comment:
We may want a min-height on this text area as the height adjuster behaves a little weird with how the pages scrolls and can easily hide the first line of text unintentionally. 

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/94493768-c10f3700-01aa-11eb-8728-6f10476dc3b4.gif" />
</p>



`min-height: 45px;` seems reasonable for one line, or we could default to `min-height: 151px;` which is what it defaults to on load.

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/94493752-b8b6fc00-01aa-11eb-8a86-1e5010acb41e.gif" />
</p>